kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -26,6 +27,11 @@ class Helm:
     def __init__(self, helm_config: HelmConfig) -> None:
         self._context = helm_config.context
         self._debug = helm_config.debug
+        self._version = self.get_version()
+        if self._version.major != 3:
+            raise RuntimeError(
+                f"The supported Helm version is 3.x.x The current Helm version is {self._version.major}.{self._version.minor}.{self._version.patch}"
```

Comment:
There is also a period missing in between the sentences 😬
```suggestion
                f"The supported Helm version is 3.x.x. The current Helm version is {self._version.major}.{self._version.minor}.{self._version.patch}"
```

Reply:
Done